test/unit/core/system.ts

```diff
@@ -21,27 +23,51 @@ suite('System', ({ expect, spy, stub }) => {
   describe('bootstrap()', () => {
     it('should transform config', () => {
       const app: any = { a: 'b' };
-      const config: any = { c: 'd' };
-      const finalConfig = { e: 'f' };
+      const finalConfig = { options: {} };
       const system = new System(app);
       const transform = stub(Configuration.Transformer, 'transform').returns(finalConfig);
       stub(fluxPkg, 'default');
 
-      system.bootstrap({}, config);
+      system.bootstrap({}, CONFIG);
 
       expect(app.config).to.eq(finalConfig);
-      expect(transform).to.be.calledWith(config);
+      expect(transform).to.be.calledWith(CONFIG);
+    });
+
+    it('should set riot version', () => {
```

Comment:
Is `app.riot` just a version number, or is it the actual `riot` instance? If the latter, perhaps "instance" is a better word.